sparkpost/django/message.py

```diff
@@ -45,9 +45,14 @@ def __init__(self, message):
                     )
 
         if message.attachments:
-            raise UnsupportedContent(
-                'The SparkPost Django email backend does not '
-                'currently support attachment.'
-            )
+            formatted['attachments'] = []
+            for attachment in message.attachments:
+                filename, content, mimetype = attachment
+                formatted['attachments'].append({
+                    'name': filename,
+                    'data': content,
+                    'type': mimetype
+                })
+            print(message.attachments)
```

Comment:
It may be little to to comment in this PR but I was just trying to understand the purpose of this file.It looks like only used in tests. In that case, should it be in tests directory or you wanted it to be usable in other places. 

Also is this `print` intentional?